src/components/ArtistName.css

```diff
@@ -0,0 +1,29 @@
+.artist-name-link:not(:first-of-type)::before {
+  content: " , ";
+  color: #a0a0a0;
+}
+
+.artist-name-link:last-child::before {
+  content: " & ";
```

Comment:
Awesome solution for this! 🙌🏽

Reply:
Thank you! Found a lot of good inpout on Technigo stack overflow